src/ColorPicker.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<widget id="com.mendix.widget.custom.colorpicker.ColorPicker" needsEntityContext="true" offlineCapable="true" xmlns="http://www.mendix.com/widget/1.0/">
+    <name>Color picker</name>
+    <description>Pick a color from color input</description>
+    <icon/>
+    <properties>
+        <property key="format" type="enumeration" defaultValue="hex">
+            <caption>Color format</caption>
+            <category>Data source</category>
+            <description>The output formatting of the selected color</description>
+            <enumerationValues>
+                <enumerationValue key="hex">hex</enumerationValue>
+                <enumerationValue key="rgb">rgb</enumerationValue>
+                <enumerationValue key="rgba">rgba</enumerationValue>
+            </enumerationValues>
+        </property>
+        <property key="colorAttribute" type="attribute">
+            <caption>Color attribute</caption>
+            <category>Data source</category>
+            <description>Attribute containing a valid HTML color: rgb(115,159,159), rgba(195,226,226,1) or #d0d0d0</description>
```

Comment:
Add `e.g.` ... Just listing the colours like that makes it look like those are the only valid colours.

Reply:
Also add the non-code values... like `red`

Reply:
The non-code values are not supported because this will affect the resulting output